### ISSUE TITLE
[Error] ImageGrab on Linux

### ISSUE THREAD

**Issue by simonecris** [wpa-2/telegram.py#10]
## Expected Behavior
Running the plugin.

## Actual Behavior
The plugin doesn't get loaded.
After a `--debug` inspection, it turns out that the `ImageGrab` module of PIL now is OSX and Windows only:

```python
from PIL import ImageGrab

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/PIL/ImageGrab.py", line 29, in <module>
    raise ImportError("ImageGrab is macOS and Windows only")
``` 

A fast and reliable workaround that worked for me is using `pyscreenshot`. 

## Specifications

  - PIL Version: 6.2.0

**Comment by wpa-2** [comment]
Just tested this on the latest image form jay and it works no issues, what OS version are you using?

**Comment by wpa-2** [comment]
![Screenshot 2024-02-03 203815](https://github.com/wpa-2/telegram.py/assets/9049886/ce81cfcb-530a-4b08-a507-598765667bcd)

**Comment by wpa-2** [comment]
> ## Expected Behavior
> Running the plugin.
> 
> ## Actual Behavior
> The plugin doesn't get loaded. After a `--debug` inspection, it turns out that the `ImageGrab` module of PIL now is OSX and Windows only:
> 
> ```python
> from PIL import ImageGrab
> 
> Traceback (most recent call last):
>   File "/usr/local/lib/python3.7/dist-packages/PIL/ImageGrab.py", line 29, in <module>
>     raise ImportError("ImageGrab is macOS and Windows only")
> ```
> 
> A fast and reliable workaround that worked for me is using `pyscreenshot`.
> 
> ## Specifications
> * PIL Version: 6.2.0

Feel free to make a PR if you have it working.

**Comment by simonecris** [comment]
> Just tested this on the latest image form jay and it works no issues, what OS version are you using?

I had problems with latest jay's (2.6.4) so I'm currently on your latest release, 1.5.5FIX.

**Comment by wpa-2** [comment]
Fixed with #11
